exercises/csharp/TriviaGame/TriviaGame/Board.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace TriviaGame
+{
+    public class Board
+    {
+        bool isGettingOutOfPenaltyBox;
+
+        private Player _player;
+        private Category _category;
+
+        public Board(Player player, Category category)
+        {
+            _player = player;
+            _category = category;
+        }
+
+        public void IsGettingOutOfPenaltyBox(int roll)
+        {
+            if (roll % 2 != 0)
+            {
+                isGettingOutOfPenaltyBox = true;
+
+                Console.WriteLine(_player.players[StaticVariables.CurrentPlayer] + " is getting out of the penalty box");
```

Comment:
The code to get a current player makes it a bit more verbose and difficult to read, would it be better to use a temporary variable for it?

Reply:
Are you referring to line 26 in the console write line?

Reply:
`_player.players[StaticVariables.CurrentPlayer]` I was thinking this variable could be pulled out into something like `var currentPlayer` to make it a bit easier to read because it's used several times

Reply:
yeah it could have been extracted out easily and would be better but for more readability